CRM/Grant/BAO/Grant.php

```diff
@@ -110,19 +110,6 @@ public static function getGrantStatusOptGroup() {
   }
 
   /**
-   * Retrieve statistics for grants.
-   *
-   *
-   * @param bool $admin
-   *
-   * @return array
-   *   Array of grant summary statistics
-   */
-  public static function getGrantStatistics($admin = FALSE) {
-    $grantStatuses = array();
-  }
```

Comment:
This stuck in skimming the PR, but it makes sense -- getGrantStatistics() doesn't do anything, and it's not called anywhere in 4.3, 4.4, 4.5 or 4.6.